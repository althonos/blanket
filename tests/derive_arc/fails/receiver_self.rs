extern crate blanket;

use blanket::blanket;

#[blanket(derive(Arc))]
pub trait Extract {
    fn extract(self);
}

fn main() {}
