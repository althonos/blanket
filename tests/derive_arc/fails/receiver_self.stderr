error: cannot derive `Arc` for a trait declaring `self` methods
 --> $DIR/receiver_self.rs:7:16
  |
7 |     fn extract(self);
  |                ^^^^
